### PR TITLE
Language translate - real time - using ngx-translate

### PR DESCRIPTION
**Pull request by mariano-romero-jd**

### CODE REVIEW

File: src/app/models/languages.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const Languages: Array<any> = [
```
Comment (angelaraya) [inline src/app/models/languages.ts:1]: This file does not belong at /models/. We can have it at the root of the project.

File: src/app/views/navbar/navbar.component.ts [hunk 2]
```diff
@@ -9,6 +9,10 @@ import { Store } from '@ngrx/store';
 export class NavbarComponent implements OnInit {
 
   @Input() webId!: string | null;
+  // @Input() languageSelect = Array<Languages>();
+  @Input() languageSelect = Languages;
+  // @Input() languageSelect = AllLanguages;
+  // @Output() selectedLanguage = new EventEmitter<Languages>();
```
Comment (angelaraya) [inline src/app/views/navbar/navbar.component.ts:15]: Why the commented out code?

**Review by angelaraya — CHANGES_REQUESTED** [review]

File: src/app/views/navbar/navbar.component.ts (hunk 2 above)
Comment (mariano-romero-jd) [reply src/app/views/navbar/navbar.component.ts]: lines removed

**Review by angelaraya — APPROVED** [review]

**Comment by elf-pavlik** [comment]
Thank you @mariano-romero-jd , do you still plan to try resolve failing CI or would you prefer that I take a look at it and add a commit to get it to pass?

**Pull request merged**